apps/api/src/controllers/applicationController.ts

```diff
@@ -0,0 +1,97 @@
+import { Request, Response } from 'express';
+import applicationService from '@/service/application-service.js';
+import { getDbInstance } from '@/db/index.js';
+import { OrderBy, type ApplicationService } from '@/service/types.js';
+import { ApplicationStateValues } from '@pcgl-daco/data-model/src/types.js';
+
+/**
+ * Handles the creation of a new application.
+ * Endpoint: POST /applications
+ */
+export const createApplicationController = async (req: Request, res: Response) => {
+  const { user_id } = req.body;
+
+  if (!user_id) {
+    return res.status(400).json({ success: false, message: 'User ID is required' });
+  }
+
+  try {
+    const database = getDbInstance();
+	const service: ApplicationService = applicationService(database);
```

Comment:
Looks like we will need to check if you have Prettier set up, this tab indent is not lined up